modules/storages/app/common/storages/peripherals/storage_error_helper.rb

```diff
@@ -42,5 +42,20 @@ def raise_error(error)
         raise API::Errors::InternalError.new
       end
     end
+
+    def error_to_code(error)
+      case error.code
+      when :bad_request
```

Comment:
You can get a similar mapping out of [HTTP Response](https://www.rubydoc.info/gems/http/HTTP/Response/Status#SYMBOLS-constant) :)

Reply:
True, nice hint, thanks. Sadly we do not use that gem (yet? :D). But I won't introduce a new dependency just because of some http code mappings. ^^